### DISCUSSION TITLE
弯曲文本检测问题

### DISCUSSION THREAD

**Discussion by outman-goutian** [PaddlePaddle/PaddleOCR#12242] [Q&A] [answered]
对于这种弯曲文本，检测框容易把其他文本放入框内，造成识别错误，有没有好的解决方法呢，在标注自己的数据集时有有没有好的解决方法呢？

![image](https://github.com/PaddlePaddle/PaddleOCR/assets/62677747/867f9e2b-3f4d-458e-a980-291b6d9119e6)

**Comment by LucaChen** [comment]
可以用DB++ 算法，采用多点标注，你这个四点标注的，肯定是框不准的

**Comment by outman-goutian** [comment, marked as answer]
谢谢，我把推理时的det_box_type改为'poly'，对于这种弯曲的可以得到解决。
